paddle/fluid/pybind/pybind.cc

```diff
@@ -818,17 +823,25 @@ All parameter, weight, gradient are variables in Paddle.
                     [](const ExecutionStrategy &self) { return self.dry_run_; },
                     [](ExecutionStrategy &self, bool dry_run) {
                       self.dry_run_ = dry_run;
-                    });
-
-  exec_strategy.def_property(
-      "use_experimental_executor",
-      [](const ExecutionStrategy &self) {
-        return self.type_ == ExecutionStrategy::kExperimental;
-      },
-      [](ExecutionStrategy &self, bool experimental) {
-        self.type_ = experimental ? ExecutionStrategy::kExperimental
-                                  : ExecutionStrategy::kDefault;
-      });
+                    })
+      .def_property(
+          "executor_type",
```

Comment:
executor_type is a too general name which cannot provide enough information for the API

Reply:
I have added some description of this enum filed.

Reply:
Using `build_strategy.enable_parallel_graph` to enable/disable the parallel graph.